### ISSUE TITLE
First launch some issues (arch aur)

### ISSUE THREAD

**Issue by ashtonx** [ShinjiruApp/Shinjiru#34]
Well there's a bit of .. wtf ;D Seems like for whatever reason it copied bunch of files from my .home directory as themes ;)

mostly junk files i was too lazy to delete ;D or some configs... 

Also seems like some files are missing, which is prolly reason there's no detection gonna look it up in a while, still figured the weird copying of themes is something you might want to look into.

here's the log
`
[2019-03-17 05:27:46.676] Debug: Created new config directory: "/home/ashton/.config/Kazakuri/Shinjiru/themes"
[2019-03-17 05:27:46.678] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/Voice 001amped.mp3"
[2019-03-17 05:27:46.678] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/art13"
[2019-03-17 05:27:46.679] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/MiniDump.dmp"
[2019-03-17 05:27:46.679] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/Pictures"
[2019-03-17 05:27:46.680] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/pdfmom-31Y26"
[2019-03-17 05:27:46.680] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/.."
[2019-03-17 05:27:46.680] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/CrashLog.txt"
[2019-03-17 05:27:46.680] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/Default"
[2019-03-17 05:27:46.680] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/Templates"
[2019-03-17 05:27:46.682] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/Safe Browsing Csd Whitelist"
[2019-03-17 05:27:46.683] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/stderr.txt"
[2019-03-17 05:27:46.683] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/status.bin"
[2019-03-17 05:27:46.683] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/onmyoji.txt"
[2019-03-17 05:27:46.683] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/sde"
[2019-03-17 05:27:46.683] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/Desktop"
[2019-03-17 05:27:46.684] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/ws"
[2019-03-17 05:27:46.684] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/status-net.txt"
[2019-03-17 05:27:46.684] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/ragnarokClickerSave.txt"
[2019-03-17 05:27:46.685] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/clickerHeroSave.txt"
[2019-03-17 05:27:46.695] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/Safe Browsing Bloom"
[2019-03-17 05:27:46.695] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/utput=onmyoji.txt"
[2019-03-17 05:27:46.695] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/Videos"
[2019-03-17 05:27:46.696] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/ckb-k95rgb.conf"
[2019-03-17 05:27:46.696] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/unicode.txt"
[2019-03-17 05:27:46.696] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/RawCallstack.txt"
[2019-03-17 05:27:46.697] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/dxdiag.txt"
[2019-03-17 05:27:46.697] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/nanodegree.png"
[2019-03-17 05:27:46.697] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/log.txt"
[2019-03-17 05:27:46.703] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/steam-552990.log"
[2019-03-17 05:27:46.703] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/utpu=onmyoji.txt"
[2019-03-17 05:27:46.704] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/discord_backup_codes.txt"
[2019-03-17 05:27:46.708] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/Safe Browsing Bloom Filter 2"
[2019-03-17 05:27:46.708] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/Public"
[2019-03-17 05:27:46.708] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/ckb-next-dev-detect-report.gz"
[2019-03-17 05:27:46.708] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/onmyoji - shikigami shards for trade"
[2019-03-17 05:27:46.708] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/Calibre Library"
[2019-03-17 05:27:46.709] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/troubleshoot.txt"
[2019-03-17 05:27:46.711] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/Safe Browsing Download"
[2019-03-17 05:27:46.711] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/gnome-meta"
[2019-03-17 05:27:46.711] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/tsw.txt"
[2019-03-17 05:27:46.711] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/."
[2019-03-17 05:27:46.712] Debug: Copying to "/home/ashton/.config/Kazakuri/Shinjiru/themes/troubleshoot-logs.txt"
[2019-03-17 05:27:46.712] Debug: Unable to open file: "/home/ashton/.config/Kazakuri/Shinjiru/themes/Default.qss"
[2019-03-17 05:27:46.757] Warning: Invalid Context= "Mimetypes" line for icon theme:  "/home/ashton/.local/share/icons/suru-plus-dark-master/mimetypes/scalable/"
[2019-03-17 05:27:46.758] Warning: Icon theme "Suru++" not found.
[2019-03-17 05:27:47.074] Debug: File not found "/usr/bin/customMediaPlayers.json"
[2019-03-17 05:27:47.087] Debug: QNetworkReplyHttpImplPrivate::_q_startOperation was called more than once QUrl("https://shinjiru.me/static/changes.json")
[2019-03-17 05:27:51.306] Warning: Invalid request: http://127.0.0.1:42069/favicon.ico
[2019-03-17 05:27:53.054] Debug: Authenticated successfully!
`

**Comment by Kazakuri** [comment]
Hm, it should only be copying the default themes from Shinjiru's [data folder](https://github.com/ShinjiruApp/Shinjiru/blob/32458b9991726682797a63f6a69bab56c6aa750e/src/paths.cpp#L68) which should be one of:
- `~/.local/share/Kazakuri/Shinjiru/theme_data`
- `/usr/local/share/Kazakuri/Shinjiru/theme_data`
- ` /usr/share/Kazakuri/Shinjiru/theme_data`

It doesn't look like any of those are being created in the [PKGBUILD](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=shinjiru-git), but the whole copying the themes over is a bit wonky to begin with.

I think the better approach would be to keep the default themes as static resources and then monitor `~/.config/Kazakuri/Shinjiru/themes` for additional ones.

**Comment by Kazakuri** [comment]
Yeah, [the code for customMediaPlayers handling](https://github.com/ShinjiruApp/Shinjiru/blob/7f7276c38dd5f5cadb330a1567b4cb71ad94ea5e/src/detection/window_enumerator.cpp#L21) really shouldn't be looking for it in the executable's path.

I'll change that up to look in `~/.config/Kazakuri/Shinjiru` as well, but it'll also have to keep looking at the executable's path unfortunately enough since some users already have their customMediaPlayer.json there.

If you get your media player of choice recognized through the customMediaPlayer file then I'll be happy to accept a PR adding it to the [default json](https://github.com/ShinjiruApp/Shinjiru/blob/32458b9991726682797a63f6a69bab56c6aa750e/res/mediaPlayers.json).

**Comment by ashtonx** [comment]
Sent pull request regarding player.
I noticed it after the fact but it seems like aur was referring to old github repo, the file copying from home might be an issue with an older version.
